deployment/live/gcp/conformance/README.md

```diff
@@ -126,3 +133,14 @@ Finally, apply the config using `terragrunt`:
 
 This should create all necessary infrastructure, and spin up a Cloud Run instance with the
 docker image you created above.
+
+### Clean up
+
+[!IMPORTANT]
+You need to run this step on your project if you want to ensure you don't get charged into perpetuity 
+for the resources we've setup. 
+
+This will delete your project!! Do not do this on a project that you didn't create expressly to run this demo.
```

Comment:
```suggestion
**This will delete your project!**
Do not do this on a project that you didn't create expressly and exclusively to run this demo.
```